### ISSUE TITLE
android support missing Quit action

### ISSUE THREAD

**Issue by Merith-TK** [fyne-io/fyne#580]
**Describe the bug**
I used `gomobile` to build and install the example appliction on the developer docs
When i tap `Quit`, not happens at all. and the theme is default to light. 

**To Reproduce**
1. install gomobile
2. paste structs example from https://fyne.io/develop/ into main.go
3. `gomobile build` and install APK to android phone

**Device (please complete the following information):**
 - OS: [e.g. Linux, iOS] Android,
 - Go version [e.g. 1.12] `go version go1.13.5 linux/amd64`
 - Fyne version [e.g. 1.1 or git SHA]
```
[developer@archlinux fyne]$ git rev-parse HEAD 
b585df2725bb31dc46e644d748648cc181714a37
```

**Comment by andydotxyz** [comment]
Apologies you are quite right - Quit is not supported in a mobile app so the function does nothing.
This is to be compliant with Apple's documentation. Do Android apps allow this function and therefore should we add it back in?

**Comment by andydotxyz** [comment]
P.S. you should not need to use "gomobile" - have we left that in the documentation somewhere? Our preferred approach is to use "fyne package" as that is what builds all different types of graphical applications.

**Comment by Merith-TK** [comment]
Ah, well android has apps that have a "quit" or exit function where it
closes the background tasks and leaves the app.

And I tried using fyne-cross the thing just crashed (opened a issue there)
and go mobile is the thing that gives results for me so..

Will take a look at the fyne binary

**Comment by Merith-TK** [comment]
And an idea around this, 
Check if the target runtime is android, if so, use an android version of the quit function.

**Comment by andydotxyz** [comment]
Sensible

**Comment by Merith-TK** [comment]
And for IOS, make it display a toast or a pop-up saying that the quit
button does not work, and in the terminal say that the quite button is not
working (stretching it here)

**Comment by andydotxyz** [comment]
Hmm, not sure about this. I would prefer to add it to the documentation only.
The user should not care about details and an iOS user would not expect a quit button so apps should not add one.

**Comment by Merith-TK** [comment]
Good point, and i would understand that.

**Comment by andydotxyz** [comment]
on release/v1.2.1 for testing

**Comment by andydotxyz** [comment]
Released in v1.2.1
